### PR TITLE
L

### PR DESCRIPTION
**Pull request by Liux-6**

### CODE REVIEW

**Comment by seisman** [comment]
You have some options:

1. Update the main branch in your own fork, then create a new branch from the main branch
2. Just delete your own fork and clone this repository directly after you accept my invitation

**Review by Liux-6 — COMMENTED** [review]
modification

File: meetings.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Current
 
-#### 2024-09-19 / LIU Xuan / 2024 Note earthquake
```
Comment (Liux-6) [inline meetings.md:7]: wrong

File: meetings.md [hunk 2]
```diff
@@ -66,6 +66,14 @@
   Geophysical Research Letters, 25(4), 571–574.
   https://doi.org/10.1029/98GL00122
 
+
+### 2024-11-7 / Liu Xuan / 2024 Noto Earthquake Rupture Process
+
+- Xu, L., Ji, C., Meng, L., Ampuero, J.-P., Yunjun, Z., Mohanna, S., & Aoki, Y. (2024). Dual-initiation ruptures in the 2024 Noto earthquake encircling a fault asperity at a swarm edge. Science, 385(6711), 871–876. https://doi.org/10.1126/science.adp0493
```
Comment (seisman) [inline meetings.md:72]:
```suggestion
- Xu, L., Ji, C., Meng, L., Ampuero, J.-P., Yunjun, Z., Mohanna, S., & Aoki, Y. (2024). 
  Dual-initiation ruptures in the 2024 Noto earthquake encircling a fault asperity at a swarm edge. 
  Science, 385(6711), 871–876. 
  https://doi.org/10.1126/science.adp0493
```

File: meetings.md [hunk 3]
```diff
@@ -66,6 +66,14 @@
...
+
+- Tohoku University, Toda, S., Stein, R., & Temblor, Inc. (2024). Intense seismic swarm punctuated by a magnitude 7.5 Japan shock. Temblor. https://doi.org/10.32858/temblor.333
```
Comment (seisman) [inline meetings.md:74]:
```suggestion
- Tohoku University, Toda, S., Stein, R., & Temblor, Inc. (2024). 
  Intense seismic swarm punctuated by a magnitude 7.5 Japan shock. 
  Temblor. 
  https://doi.org/10.32858/temblor.333
```

**Pull request merged**